### PR TITLE
Incompatible browser detection

### PR DESCRIPTION
**Pull request by BetaPictoris**
Adds messages to detect if JavaScript and WASM are supported and enabled. 

Fixes: #636

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -43,6 +43,24 @@
...
+		LLDAP requires JavaScript, please switch to a compatible browser or enable 
+		it.
+	</noscript>
+
+	<script>
+		/* Detect if the user has WASM support. */
+		if (typeof WebAssembly === 'undefined') {
+			const pWASMMsg = document.createElement("p")
+			pWASMMsg.innerHTML = `
+			LLDAP requires WASM, please switch to a compatible browser or enable it.
```
Comment (nitnelave) [inline app/index.html:59]: For the users of brave (or derived browsers), it might require enabling JIT for JavaScript. Could we add a note about that?

**Review by nitnelave — CHANGES_REQUESTED** [review]
Looks good! Just a little note about the message.

File: app/index.html (hunk 1 above)
Comment (BetaPictoris) [reply app/index.html]: I've added JIT to the message.

**Review by nitnelave — APPROVED** [review]

**Comment by nitnelave** [comment]
Thanks!

**Comment by nitnelave** [comment]
Thank you everyone for the contribution!
This feature is now available in the latest release, [v0.5.0](https://github.com/lldap/lldap/releases/tag/v0.5.0).
You can support LLDAP by starring our repo, contributing some configuration examples and becoming a sponsor.

**Pull request merged**